src/db.rs

```diff
@@ -470,6 +470,18 @@ impl Db {
     pub fn space_amplification(&self) -> Result<f64> {
         self.context.pagecache.space_amplification()
     }
+
+    /// Returns a true value if one of the tree names linked
+    /// to the database is found, if not a false will be returned.
+    pub fn contains_tree<V: AsRef<[u8]>>(&self, name: V) -> Result<bool> {
```

Comment:
Isn't this `Result` unnecessary?

Reply:
Yeah you're right, I think I'm going to send out a new PR to work on that. I think it is because I literally copy-pasted the feature method.